backup_tests/docker_backup_tests.sh

```diff
@@ -1,27 +1,33 @@
 #!/bin/bash
 
-##########################################################################
-# Created By Manish Chawla, Percona LLC                                  #
-# This script tests backup for docker containers                         #
-# Usage:                                                                 #
-# 1. Run the script as: ./docker_backup_tests.sh pxb24/pxb8 main/testing #
-# 3. Logs are available in: $PWD/backup_log                              #
-##########################################################################
+#################################################################################
+# Created By Manish Chawla, Percona LLC                                         #
+# Modified By Mohit Joshi, Percona LLC                                          #
+# This script tests backup for docker containers                                #
+# Usage:                                                                        #
+# 1. Run the script as: ./docker_backup_tests.sh pxb24/pxb80/pxb81 main/testing #
+# 3. Logs are available in: $PWD/backup_log                                     #
+#################################################################################
 
 if [ "$#" -ne 2 ]; then
-    echo "Please run the script with parameters: <version as pxb24/pxb8> <repo as main/testing>"
+    echo "Please run the script with parameters: <version as pxb24/pxb80/pxb81> <repo as main/testing>"
     echo "Main repo is the percona docker image and testing repo is the perconalab docker image"
     exit 1
 fi
 
 clean_setup() {
     # This function checks and cleans the setup
 
-    if [ "$1" = "pxb8" ]; then
+    if [ "$version" = "pxb80" ]; then
         if [ "$(sudo docker ps -a | grep mysql-8.0)" ]; then
             sudo docker stop mysql-8.0 >/dev/null 2>&1
             sudo docker rm mysql-8.0 >/dev/null 2>&1
         fi
+    elif [ "$version" = "pxb81" ]; then
+        if [ "$(sudo docker ps -a | grep mysql-8.1)" ]; then
+            sudo docker stop mysql-8.1 >/dev/null 2>&1
```

Comment:
Whi don't we use $server here?

Reply:
This is because the variable "$server" is set inside the function test_pxb8_docker(). We cannot use this variable before this function is called. 
As we are calling the function clean_setup() before calling the function test_pxb8_docker(), this variable will not be set.

Reply:
Then maybe it makes sense to move vars definition from this function to main part of the script? $server defines the name of docker container that we have created and clean_setup() should remove exactly this container.

Reply:
Maybe I am missing smth but this does not seem as many changes and this is more for consistency. We make clean_setup() to make sure that when test_pxb8_docker() is run there will not be errors that the container with this name exists. What if at some point we need to change name for server="mysql-8.0" to server="mysql-8.0.x" or we need to add one more server. Smb can miss updating clean_setup()  manually to change/add `if` for clean_setup(). This is my concern. We can leave it as is for now (this is not critical update) but need to consider it for future in addition to changing ifs to variables as it was mentioned in comment to PR itself.